src/main/features/claim/routes/fees.ts

```diff
@@ -8,43 +8,52 @@ import { RangeGroup } from 'app/fees/models/rangeGroup'
 import { FeesTableViewHelper } from 'claim/helpers/feesTableViewHelper'
 import { DraftClaim } from 'drafts/models/draftClaim'
 import { Draft } from '@hmcts/draft-store-client'
+import { ClaimAmountExceedsLimitError } from '../../../errors'
 
 const supportedFeeLimitInPennies: number = 1000000
 
 /* tslint:disable:no-default-export */
 export default express.Router()
-  .get(Paths.feesPage.uri, async (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> => {
-    const draft: Draft<DraftClaim> = res.locals.claimDraft
-    const claimAmount: number = await claimAmountWithInterest(draft.document)
-    Promise.all(
-      [
-        FeesClient.calculateIssueFee(claimAmount),
-        FeesClient.calculateHearingFee(claimAmount),
-        FeesClient.getIssueFeeRangeGroup(),
-        FeesClient.getHearingFeeRangeGroup()
-      ]
-    )
-      .then((values: any[]) => {
-        const issueFeeRangeGroup: RangeGroup = values[2]
-        const hearingFeeRangeGroup: RangeGroup = values[3]
+  .get(Paths.feesPage.uri,
```

Comment:
Please add route test that reflects scenario you are fixing. In test case draft should have an amount set to 10k and interest declared from some date in the past that will result in final amount being over 10k.

REQ

Reply:
done